src/shared/layouts/BaseLayout/BaseLayout.tsx

```diff
@@ -7,9 +7,9 @@ import {
   TransferOwnershipPopup,
 } from 'modules/Dashboard/features/Applet/Popups';
 import { useAppDispatch } from 'redux/store';
-import { auth } from 'modules/Auth/state';
-import { account, folders, popups, users } from 'modules/Dashboard/state';
-import { Footer } from 'shared/components';
+import { account, folders, popups, workspaces, applets, users, auth } from 'redux/modules';
+import { DEFAULT_ROWS_PER_PAGE, Footer } from '../..//components';
```

Comment:
could you pls use full routes for '../..//components' and  '../../types'?

Reply:
let's keep it, to avoid circular dep

Reply:
as I think circular diapedesis occur after importing from one directory, for example shared/components
if we have different: shared/components and shared/layouts there are no circular deps